commit 679a0fd5436a8ef825c15407ff88def43b68a81b
Author: Felix Fietkau <nbd@nbd.name>
Date:   Mon Mar 6 10:04:25 2017 +0100

    ubifs: Fix RENAME_WHITEOUT support
    
    commit c3d9fda688742c06e89aa1f0f8fd943fc11468cb upstream.
    
    Remove faulty leftover check in do_rename(), apparently introduced in a
    merge that combined whiteout support changes with commit f03b8ad8d386
    ("fs: support RENAME_NOREPLACE for local filesystems")
    
    Fixes: f03b8ad8d386 ("fs: support RENAME_NOREPLACE for local filesystems")
    Fixes: 9e0a1fff8db5 ("ubifs: Implement RENAME_WHITEOUT")
    Signed-off-by: Felix Fietkau <nbd@nbd.name>
    Signed-off-by: Richard Weinberger <richard@nod.at>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1506_679a0fd5436a8ef825c15407ff88def43b68a81b.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1506_679a0fd5436a8ef825c15407ff88def43b68a81b.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
